scripts/build-windows/kcl-doc.go

```diff
@@ -6,54 +6,85 @@
 package main
 
 import (
+	"fmt"
 	"os"
+	"os/exec"
 	"path/filepath"
-	"syscall"
-	"unsafe"
 )
 
 func main() {
-	// kclvm -m kclvm ...
+	// python3 -m kclvm ...
+
 	var args []string
-	args = append(args, os.Args[0])
-	args = append(args, "-m", "kclvm.tools.docs")
+	args = append(args, "/C", "python3", "-m", "kclvm.tools.docs")
 	args = append(args, os.Args[1:]...)
+
 	os.Exit(Py_Main(args))
 }
 
-var (
-	python39_dll = syscall.NewLazyDLL(findKclvm_dllPath())
-	proc_Py_Main = python39_dll.NewProc("Py_Main")
-)
-
-// int Py_Main(int argc, wchar_t **argv)
 func Py_Main(args []string) int {
-	c_args := make([]*uint16, len(args)+1)
-	for i, s := range args {
-		c_args[i] = syscall.StringToUTF16Ptr(s)
+	inputPath, err := os.Executable()
+	if err != nil {
+		fmt.Println("Input path does not exist")
```

Comment:
Should the promotion information be output to stderr?

Reply:
fixed